doc/documentation.rst

```diff
@@ -72,36 +72,36 @@ be named ``foo.mld``, ``bar.mld``, and ``baz.mld``
     (package foo)
      (mld_files foo bar baz))
 
-This stanza will attach all mld files excluding ``wip.mld`` in the current
-directory to the inferred package:
+This stanza will attach all ``.mld`` files to the inferred package, 
+excluding ``wip.mld``, in the current directory:
 
 .. code-block:: lisp
 
    (documentation
     (mld_files :standard \ wip))
 
-All mld files attached to a package will be included in the generated
-``.install`` file for that package, and hence will be installed by opam.
+All ``.mld`` files attached to a package will be included in the generated
+``.install`` file for that package. They'll be installed by Opam.
 
-Package entry page
+Package Entry Page
 ------------------
 
 The ``index.mld`` file (specified as ``index`` in ``mld_files``) is
-treated specially by dune. This will be the file used to generate the
+treated specially by Dune. This will be the file used to generate the
 entry page for the package, linked from the main package listing.
 
 To generate pleasant documentation, we recommend writing an
-``index.mld`` file with at least short description of your package,
+``index.mld`` file with at least short description of your package 
 and possibly some examples.
 
-If you do not write your own ``index.mld`` file, dune will generate
+If you do not write your own ``index.mld`` file, Dune will generate
 one with the entry modules for your package. But this generated file
 will not be installed.
 
 .. _odoc-options:
 
-Passing options to Odoc
-=======================
+Passing Options to ``odoc``
```

Comment:
Isn't it weird that we're referring to the Odoc tool by the name of its binary?

Reply:
It is very weird, yes. It's how Jon wants it written, though.

Reply:
@jonludlam it's still weird, but I'm merging.